src/options/smt_options.toml

```diff
@@ -245,6 +245,7 @@ name   = "SMT Layer"
   long       = "produce-assertions"
   type       = "bool"
   alias      = ["interactive-mode"]
+  default    = "true"
   help       = "keep an assertions list (enables get-assertions command)"
```

Comment:
Can we change the help message to say that this is always on and changing it has no effect?